package/gluon-core/luasrc/lib/gluon/upgrade/002-ath79-migration

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/lua
+
+local platform = require 'gluon.platform'
+local sysconfig = require 'gluon.sysconfig'
+
+local unistd = require 'posix.unistd'
+local uci = require('simple-uci').cursor()
+
+
+if not platform.match('ath79') then
+    return
+end
+
+if platform.match('ath79', 'generic', {'devolo,dvl1200e', 'devolo,dvl1200i',
+                                           'devolo,dvl1750e', 'devolo,dvl1750c',
+                                           'devolo,dvl1750i', 'glinet,gl-ar300m-lite',
+                                           'glinet,gl-ar750s', 'ocedo,raccoon',
+                                           'tplink,archer-c6-v2', 'tplink,cpe220-v3'}) then
+    return
+end
+
+if not sysconfig.gluon_version then
+    os.execute('touch /lib/gluon/core/sysconfig/migrations/ar71xx-ath79')
+    return
+end
+
+if unistd.access('/lib/gluon/core/sysconfig/migrations/ar71xx-ath79') then
+    return
+end
+
+os.execute('rm /etc/board.json')
+os.execute('board_detect')
+
+-- Remove LED configuration
+uci:foreach('system', 'rssid', function(config)
+    uci:delete('system', config['.name'])
+end)
+uci:foreach('system', 'led', function(config)
+    uci:delete('system', config['.name'])
+end)
+uci:commit('led')
+
+-- Create LED configuration
+os.execute('/lib/gluon/core/migration/ath79-create-config.sh')
+
+local function swap_eth0_eth1()
```

Comment:
unused?

Reply:
Yes, It's not needed for this specific board, but included for future use.